.gitignore

```diff
@@ -1 +1,2 @@
-myvenv
\ No newline at end of file
+myvenv
+*.pyc
```

Comment:
Hmm, why we still see `.pyc` files in diff :D? You can add whole `__pycache__` folders maybe? Also database :)